app/models/decidim/solutions/solution.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Solutions
+    # The data store for a Solution in the Decidim::Solutions component.
+    class Solution < Challenges::ApplicationRecord
+      include Decidim::HasComponent
+      include Decidim::Loggable
+      include Decidim::Publicable
+      include Decidim::Resourceable
+      include Decidim::ScopableComponent
+      include Decidim::Searchable
+      include Decidim::Traceable
+      include Decidim::TranslatableAttributes
+
+      # FIX define possible requirements !
+      VALID_REQUIREMENTS = [:technical, :economic, :financial].freeze
+      enum requirements: VALID_REQUIREMENTS
```

Comment:
This is not an enum but a free text field

Reply:
oh, ok @tramuntanal ! I'll change it then!